Tests/OpenAPIRuntimeTests/Base/Test_OpenAPIValue.swift

```diff
@@ -193,4 +193,96 @@ final class Test_OpenAPIValue: Test_Runtime {
         XCTAssertEqual(value[0] as? String, "one")
         XCTAssertEqual(value[1] as? [String: Int], ["two": 2])
     }
+
+    func testEncoding_objectNested_success() throws {
+
+        struct Foo: Encodable {
+            var bar: String
+            var dict: OpenAPIObjectContainer = .init()
+        }
+
+        do {
+            let value = Foo(
+                bar: "hi",
+                dict: try .init(unvalidatedValue: [
+                    "baz": "bar",
+                    "number": 1,
+                    "nestedArray": [
+                        1,
+                        [
+                            "k": "v"
+                        ],
+                    ] as [(any Sendable)?],
+                    "nestedDict": [
+                        "nested": 2
+                    ],
+                ])
+            )
+            let encoder: JSONEncoder = .init()
+            encoder.outputFormatting = [.prettyPrinted, .sortedKeys]
+            let data = try encoder.encode(value)
+            XCTAssertEqual(
```

Comment:
Please make it consistent with the other tests by using the utility functions that does the encoding/decoding and equality checks.